src/lib.rs

```diff
@@ -2729,6 +2731,68 @@ pub fn is_maximal_matching(
     })
 }
 
+/// Find the edges in the minimum spanning tree or forest of a graph
+/// using Kruskal's algorithm.
+///
+/// :param PyGraph graph: Undirected weighted graph with comparable weights
```

Comment:
I would probably expand this a bit as it's a unique constraint for a retworkx function, something like: "python comparisons (ie `>` or `<`)  are used to compare the weights of edges. Ensure that the objects on the input graph are types that can be compared in this way."